docs/get-started.md

```diff
@@ -1,42 +1,120 @@
-- Azure CLI
+- [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
-### Set env vars
-The OSM Azure repo relies on environment variables to make it usable on your localhost. The root of the repository contains a file named `.env.example`. Copy the contents of this file into `.env`
+## Add Azure CLI extensions
+
+> Note: Get latest wheels: https://github.com/Azure/azure-arc-kubernetes-preview/tree/master/extensions
+
+1. Install the latest [connectedk8s](https://docs.microsoft.com/en-us/cli/azure/connectedk8s?view=azure-cli-latest).
 ```bash
-cat .env.example > .env
+WHEEL="https://arcextensions.blob.core.windows.net/extensions/connectedk8s-0.3.8-py2.py3-none-any.whl"
+az extension add --source $WHEEL --yes || true
 ```
-The various environment variables are documented in the `.env.example` file itself. Modify the variables in `.env` to suite your environment.
 
-Note: Ensure the region specified in the .env file has the latest OSM arc extension available. Supported regions listed [here](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/connect-cluster#supported-regions).
+2. Install the latest [k8s-extension](https://docs.microsoft.com/en-us/cli/azure/k8s-extension?view=azure-cli-latest).
+```bash
+WHEEL="https://arcextensions.blob.core.windows.net/extensions/k8s_extension-0.1PP.13-py2.py3-none-any.whl"
+az extension add --source $WHEEL --yes || true
+```
 
+## Enable Arc Features
+The following steps need to be run only once for a given subscription:
 
-### Enable Arc Features
-This step needs to be run only once for every new subscription.
-```sh
-./scripts/enable-arc-feature.sh
+_Optional_: set default Azure subscription for the `az` CLI:
+```bash
+az account set --subscription=<your-Azure-subscription-ID>
 ```
 
-### Add Arc Extension to Cluster
-This step needs to be run for each Kubernetes cluster that needs the Arc OSM extension. It will add the connectedk8s extension to the Azure CLI, enable arc on the Kubernetes cluster (installs arc agents and components), and install the osm arc extension on the Kubernetes cluster.
-```sh
-./scripts/add-extension.sh
+```bash
+az feature register \
+  --namespace Microsoft.Kubernetes \
+  --name previewAccess
 ```
 
-### Remove Arc Extension from Cluster
-This step needs to be run for each cluster when you want to remove the OSM Arc extension. Note that this script does not remove Arc connectivity.
+```bash
+az feature register \
+  --namespace Microsoft.KubernetesConfiguration \
+  --name sourceControlConfiguration
+```
 
-This script also does not delete the `arc-osm-system` and `azure-arc` namespaces.
-```sh
-./scripts/delete-extension.sh
+```bash
+az feature register \
+  --namespace Microsoft.KubernetesConfiguration \
+  --name extensions
+```
+
+```bash
+az provider register \
+  --namespace Microsoft.ExtendedLocation
+```
+
+```bash
+az feature register \
+  --namespace Microsoft.ExtendedLocation \
+  --name CustomLocations-ppauto
+```
+
```

Comment:
Another option instead of writing out all the commands here is to direct the developer to the scripts in the ./scripts folder, such as add-extension.sh, connect-to-arc.sh, install-connectedk8s.sh, and install-k8s-extension.sh

Reply:
I don't find documents that point to files with bash scripts to be a great experience.
Document should lay out the steps it takes to get something done.  The document itself is not the automation tool. The automation tool could be influenced by the document, but not the other way around.